examples/basic/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@electron-webpack-examples/basic",
```

Comment:
Nice! I always like when there are examples included

Reply:
Nitpick, but should this be `@electron-redux-webpack-examples/basic`?